auditors/capabilities/capabilities.go

```diff
@@ -73,45 +60,70 @@ func getOverrideLabel(capability string) string {
 	return overrideLabelPrefix + strings.Replace(strings.ToLower(capability), "_", "-", -1)
 }
 
-func auditContainerForCapability(container *k8stypes.ContainerV1, capability string, dropList []string) (auditResults []*kubeaudit.AuditResult) {
-	if isCapabilityAdded(container, capability) {
-		auditResult := &kubeaudit.AuditResult{
-			Name:     CapabilityAdded,
-			Severity: kubeaudit.Error,
-			Message:  fmt.Sprintf("Capability added. It should be removed from the capability add list. If you need this capability, add an override label such as '%s: SomeReason'.", override.GetContainerOverrideLabel(container.Name, getOverrideLabel(capability))),
-			PendingFix: &fixCapabilityAdded{
-				container:  container,
-				capability: capability,
-			},
-			Metadata: kubeaudit.Metadata{
-				"Container":  container.Name,
-				"Capability": capability,
-			},
-		}
-		auditResults = append(auditResults, auditResult)
+func auditContainer(container *k8stypes.ContainerV1, capability string, addList []string) []*kubeaudit.AuditResult {
+	var auditResults []*kubeaudit.AuditResult
+
+	if isCapabilityInArray(capability, addList) {
+		return auditResults
 	}
 
-	if isCapabilityNotDropped(container, capability, dropList) {
+	if SecurityContextOrCapabilities(container) {
+		var message string
+
+		if IsCapabilityInAddList(container, capability) {
+			message = fmt.Sprintf("Capability added. It should be removed from the capability add list. If you need this capability, add an override label such as '%s: SomeReason'.", override.GetContainerOverrideLabel(container.Name, getOverrideLabel(capability)))
```

Comment:
Would it be possible to specify the capability that is Added? Something like:
```suggestion
			message = fmt.Sprintf("Capability \"%s\" added. It should be removed from the capability add list. If you need this capability, add an override label such as '%s: SomeReason'.", capability, override.GetContainerOverrideLabel(container.Name, getOverrideLabel(capability)))
```